filebeat/input/file/state.go

```diff
@@ -10,6 +10,7 @@ import (
 
 // State is used to communicate the reading state of a file
 type State struct {
+	Id          string        `json:"-"` // local unique id to make comparison more efficient
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
struct field Id should be ID